packages/react/src/menu-item/MenuItem.tsx

```diff
@@ -0,0 +1,58 @@
+import * as RadixMenu from "@radix-ui/react-dropdown-menu";
+import { type ReactNode, forwardRef } from "react";
+
+import { Box, type BoxProps } from "../box";
+import { Flex } from "../flex";
+import { extractSprinkles } from "../sprinkles";
+import { Text } from "../text";
+import * as styles from "./MenuItem.css";
+
+type MenuItemProps = BoxProps<
+  typeof RadixMenu.Item,
+  {
+    endDecorator?: ReactNode;
+    startDecorator?: ReactNode;
+  }
+>;
+
+export const MenuItem = forwardRef<HTMLDivElement, MenuItemProps>(
+  ({ children, disabled, endDecorator, startDecorator, ...props }, ref) => {
+    const { restProps, sprinkleProps } = extractSprinkles(props);
+
+    return (
+      <Flex {...styles.itemRoot()}>
+        <Box asChild ref={ref} {...sprinkleProps}>
+          <RadixMenu.Item disabled={disabled} {...restProps}>
+            <Flex {...styles.item()}>
```

Comment:
As we are spreading props, we need not to handle `disabled` explicitly. We can also remove it from parameters.
```suggestion
          <RadixMenu.Item {...restProps}>
```